api/agent/drivers/docker/cookie.go

```diff
@@ -356,7 +356,7 @@ func (c *cookie) PullImage(ctx context.Context) error {
 
 	err := c.drv.docker.PullImage(docker.PullImageOptions{Repository: repo, Tag: c.imgTag, Context: ctx}, *cfg)
 	if err != nil {
-		log.WithError(err).Error("Failed to pull image")
+		log.WithError(err).Info("Failed to pull image")
```

Comment:
```suggestion
		log.WithError(err).Debug("Failed to pull image")
```
I'm not sure this even wants to be an Info level.

Reply:
Actually I see this was already discussed.